site/_posts/2022-11-01-authenticators-ceremonies-webauthn-oh-my.md

```diff
@@ -0,0 +1,128 @@
+6. The server verifies the signature of the unique challenge that it sent to the user (in step 2). Upon successful verification, the user is logged into the web application.
+
+## WebAuthn vs. FIDO 2
+
+WebAuthn was developed by the [World Wide Web Consortium (W3C)](http://www.w3.org/), which is the international standards organization for the world wide web. Meanwhile, Fast Identity Online 2 (FIDO2) was created by [FIDO Alliance](https://fidoalliance.org/) with the aim of creating passwordless authentication. WebAuthn is a core component of [FIDO2 specifications](https://fidoalliance.org/fido2/).
+
+The FIDO Alliance decided to partner with W3C to make FIDO2 authentication a standard for the web. WebAuthn became a part of the FIDO2 specification, and a [set of browser APIs](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API) was finalized, which is widely used today for passwordless authentication.
+
+## Common WebAuthn Terminology
+
+If you're looking to enter into the WebAuthn space, it can be helpful to understand the following terminology:
+
+### Cross-Platform vs. Single-Platform
+
+[Cross-platform authenticators](https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Platform_vs_Cross-Platform.html.) (also known as roaming authenticators) are authenticators that can be used with another device, like a [YubiKey](https://www.yubico.com/), to perform authentication.
+
+[Single-platform authenticators](https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Platform_vs_Cross-Platform.html) (also known as platform authenticators) are authenticators that are built into the device, like Android pattern gestures or biometrics, iOS Face ID, or voice unlocks.
+
+### User Presence vs. User Verification 
+
+User presence indicates the user's current state (*ie* whether someone is present). For instance, an authenticator may have a button a person pushes to indicate they are in possession of it. In this situation, a user is present, but as a relying party, you don't know which user. 
+
+On the other hand, user verification is concerned with the authenticity of the user and verifies that the user is authenticated, for example by requiring a pin or facial recognition before they are allowed access.
+
+### First Factor vs. Second Factor
+
+First factor authentication means that the user is identified and authenticated. This is often done with a username and password. The username identifies the user and the password authenticates them. 
+
+A second factor authentication uses another factor for authentication. This second factor doesn't need to identify the user, since that was accomplished with the first factor. By requiring a second factor of authentication, the authenticating system is more certain of who is authenticating. A typical example is when a user tries to login to an application, and then gets an OTP sent to their email which they can use to complete authentication.
```

Comment:
```suggestion
A second factor authentication uses another factor for authentication. This second factor doesn't need to identify the user, since that was accomplished with the first factor. By requiring a second factor of authentication, the authenticating system is more certain of who is authenticating. A typical example is when a user tries to log in to an application, and then gets an OTP sent to their email which they can use to complete authentication.
```